.github/workflows/odh-manifests-PR-sync.yml

```diff
@@ -0,0 +1,70 @@
+name: odh-manifests sync
+
+run-name: Sync manifests in odh-manifests
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout data-science-pipelines-operator repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: main
+          repository: DharmitD/data-science-pipelines-operator
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Send pull-request
+        run: |
+          LATEST_TAG=$(git describe --tags --always --abbrev=0)
+          REPOSITORY="DharmitD/odh-manifests"
```

Comment:
This repo doesn't seem right

Reply:
Done, updated repo to `opendatahub-io/odh-manifests`